pom.xml

```diff
@@ -96,6 +96,34 @@ under the License.
     <surefire.version>3.1.0</surefire.version><!-- for surefire, failsafe and surefire-report -->
     <maven.plugin.tools.version>3.9.0</maven.plugin.tools.version><!-- for m-plugin-p and maven-plugin-annotations -->
     <assembly.tarLongFileMode>posix</assembly.tarLongFileMode>
+
+    <!-- plugins versions -->
+    <apacheRatPluginVersion>0.15</apacheRatPluginVersion>
+    <mavenAntrunPluginVersion>3.1.0</mavenAntrunPluginVersion>
+    <mavenAssemblyPluginVersion>3.6.0</mavenAssemblyPluginVersion>
```

Comment:
Please stay consistent in Naming: `surefire.version` != `mavenAssemblyPluginVersion`.

If velocity has problems, let's fix velocity. In freemarker we could just use another syntax, I expect velocity to have the same ability. 3rd party tools (ie non-maven) should not have impact on naming.